src/modules/authz/help.tsx

```diff
@@ -0,0 +1,114 @@
+  deleteDropBox,
+  deleteProject,
+  deleteReferenceMaterial,
+  downloadData,
+  editDataset,
+  editPermissions,
+  editProject,
+  exportData,
+  ingestData,
+  ingestDropBox,
+  ingestReferenceMaterial,
+  queryData,
+  queryDatasetLevelBoolean,
+  queryDatasetLevelCounts,
+  queryProjectLevelBoolean,
+  queryProjectLevelCounts,
+  viewDropBox,
+  viewNotifications,
+  viewPermissions,
+  viewRuns,
+} from "bento-auth-js";
+
+export const PERMISSIONS_HELP: Record<string, ReactNode> = {
+  // data
+  [queryData]: "Whether the subject can access data records for the resource, e.g. phenotypic metadata, experiments.",
+  [downloadData]:
+    "Whether the subject can download data files associated with the resource, e.g., download VCFs and other " +
+    "experiment results.",
+  [deleteData]: "Whether the subject can delete data from the resource, e.g., clearing all variants.",
+  [ingestData]: "Whether the subject can ingest new data into the resource, e.g., adding new biosamples.",
+  [analyzeData]: <strong>CURRENTLY UNUSED.</strong>,
+  [exportData]: <strong>CURRENTLY UNUSED.</strong>,
+
+  // dataset
+  [editDataset]:
+    "Whether the subject can edit datasets (title, description, provenance metadata) in the specified node/project " +
+    "resource.",
+  [createDataset]: "Whether the subject can create datasets in the specified node/project resource.",
+  [deleteDataset]:
+    "Whether the subject can delete datasets from the specified node/project resource. This in turn deletes data " +
+    "inside the dataset.",
+
+  // dataset_level_boolean
+  [queryDatasetLevelBoolean]:
+    "Whether the subject can see low-count-censored yes/no answers about the data at the dataset level. The " +
+    "low-count threshold is controlled by the resource's discovery configuration file.",
+
+  // dataset_level_counts
+  [queryDatasetLevelCounts]:
+    "Whether the subject can see low-count-censored count answers about the data at the dataset level. The low-count " +
+    "threshold is controlled by the resource's discovery configuration file.",
+
+  // drop_box
+  [viewDropBox]:
+    "Whether the subject can see the instance-wide drop box (staging area) for files. This permission is only valid " +
+    "for the Everything resource.",
+  [ingestDropBox]: "Whether the subject can upload files / create folders in the drop box.",
+  [deleteDropBox]: "Whether the subject can delete files / folders from the drop box.",
+
+  // notifications
+  [viewNotifications]:
+    "Whether the subject can view notifications. Currently, this only works on the instance level; any " +
+    "project/dataset context is ignored.",
+  [createNotifications]: <strong>CURRENTLY UNUSED.</strong>,
+
+  // permissions
+  [viewPermissions]:
+    "Whether the subject can view permissions which apply to only this resource, or any sub-resources.",
+  [editPermissions]:
+    "Whether the subject can edit permissions which apply to only this resource, or any sub-resources.",
```

Comment:
"only this resource, or any sub-resources"..... so, anything? I wasn't sure what this ruled out.